Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -445,5 +445,8 @@ public interface IGitModule
         string RenameBranch(string name, string newName);
 
         GitBlame Blame(string? fileName, string from, Encoding encoding, string? lines = null, CancellationToken cancellationToken = default);
+
+        IReadOnlyList<GitItemStatus>? GetGrepFilesStatus(GitRevision revision, string greps, CancellationToken cancellationToken = default);
```

Comment:
(All important Forms call this from a background task.)

Reply:
what do you mean?

Reply:
>> (All important Forms call this from a background task.)

> what do you mean?

This function does not need to be made `async`, (which would make its implementation more complicate.)

(In the "Files changed" tab, this function is located right above an added `async` function. The comment loses this context.)